distributed/client.py

```diff
@@ -1288,6 +1288,43 @@ def run(self, function, *args, **kwargs):
         """
         return sync(self.loop, self._run, function, *args, **kwargs)
 
+    @gen.coroutine
+    def _run_coroutine(self, function, *args, **kwargs):
+        workers = kwargs.pop('workers', None)
+        responses = yield self.scheduler.broadcast(msg=dict(op='run_coroutine',
+                                                function=dumps(function),
+                                                args=dumps(args),
+                                                kwargs=dumps(kwargs)),
+                                                workers=workers)
+        results = {}
+        for key, resp in responses.items():
+            if resp['status'] == 'OK':
+                results[key] = resp['result']
+            elif resp['status'] == 'error':
+                raise loads(resp['exception'])
```

Comment:
I didn't know tracebacks were shipped. Generally they aren't very picklable, but perhaps cloudpickle has special support for them?